build.gradle

```diff
@@ -21,18 +21,20 @@ apply plugin: 'nebula.rxjava-project'
 sourceCompatibility = JavaVersion.VERSION_1_6
 targetCompatibility = JavaVersion.VERSION_1_6
 
+def rsVersion = '1.0.1-RC2'
```

Comment:
@akarnokd is it possible to change name of your branch, it's name is now a bit misleading ;-) 

I mean just removing `_RC1`.